tests/test_auto_detection.py

```diff
@@ -37,19 +40,25 @@ def test_loop_auto():
     assert isinstance(policy, asyncio.events.BaseDefaultEventLoopPolicy)
     expected_loop = "asyncio" if uvloop is None else "uvloop"
     assert type(policy).__module__.startswith(expected_loop)
-    loop = asyncio.get_event_loop()
-    loop.close()
+
+    async def get_loop():
+        return anyio.lowlevel.current_token()
+
+    loop = anyio.run(get_loop)
+    assert loop.__module__.startswith(expected_loop)
```

Comment:
what about removing all that entirely, we just test the policy here, no loop is started if I'm correct and there should be no need to close it

Reply:
Yeah we could just check the policy